src/replica/disk_cleaner.cpp

```diff
@@ -70,6 +75,91 @@ const std::string kFolderSuffixBak = ".bak";
 const std::string kFolderSuffixOri = ".ori";
 const std::string kFolderSuffixTmp = ".tmp";
 
+namespace {
+
+bool get_expiration_seconds_by_last_write_time(const std::string &path,
```

Comment:
Instead of `seconds`, maybe `timestamp` would be more meaningful.

Reply:
OK, I'll replace all "expiration seconds” with "timestamp".